src/compas_view2/objects/object.py

```diff
@@ -34,3 +64,50 @@ def create(self):
 
     def edit(self):
         pass
+
+    @property
+    def translation(self):
+        return self._translation
+
+    @translation.setter
+    def translation(self, data):
+        self._translation.data = data
+
+    @property
+    def rotation(self):
+        return self._rotation
+
+    @rotation.setter
+    def rotation(self, data):
+        self._rotation.data = data
```

Comment:
i don't understand how this works with the `_update_matrix` function.
what is `self._rotation`? the update function seems to suggest that this is a triplet of angles, but here it suggests that it is a compas (transformation?) object.
same for the translation and scale.

Reply:
should it not be something like
```
@rotation.setter
def rotation(self, angles):
    assert len(angles) == 3  # not sure if this is necessary and/or makes sense
    self._rotation = angles